scapy/contrib/gtp_v2.py

```diff
@@ -1454,10 +1479,8 @@ class IE_PrivateExtension(gtp.IE_Base):
         BitField("SPARE", 0, 4),
         BitField("instance", 0, 4),
         ShortEnumField("enterprisenum", None, IANA_ENTERPRISE_NUMBERS),
-    ]
-
-    def extract_padding(self, s):
```

Comment:
Why are you removing this?

Reply:
Because, it is not able to decode the private extension IEs that I have (from production deployments)